.github/workflows/gatk-tests.yml

```diff
@@ -0,0 +1,311 @@
+name: Build and Test GATK 4
+on:
+  pull_request:
+  workflow_dispatch:
+
+env:
+  CROMWELL_VERSION: 51
+  TERM: dumb
+  GRADLE_OPTS: "-Xmx2048m -Dorg.gradle.daemon=false"
+  HELLBENDER_TEST_INPUTS: gs://hellbender/test/resources/
+  HELLBENDER_TEST_STAGING: gs://hellbender-test-logs/staging/
+  HELLBENDER_TEST_LOGS: /hellbender-test-logs/build_reports/
+  HELLBENDER_TEST_PROJECT: broad-dsde-dev
+  TEST_VERBOSITY: minimal
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  buildDocker:
+    name: build docker images
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout gatk
+        uses: actions/checkout@v2
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          tags: |
+            type=raw,value=${{github.sha}}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+        with:
+          context: .
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+
+#Run our non-docker tests
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ 8, 11 ]
```

Comment:
Maybe we should just explicitly include the ones we ant instead of the cool but confusing excludes clauses?